src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -171,6 +170,28 @@ trait MarathonConf
     true
   }
 
+  private[this] def deriveDefaultAcceptedResourceRolesDefaultBehavior: Option[AcceptedResourceRolesDefaultBehavior] = {
```

Comment:
Lol, how default is `deriveDefaultAcceptedResourceRolesDefaultBehavior`?  

/cc @ANeumann82 reminds me about `GameEventManagerManager`

Reply:
lol